index.html

```diff
@@ -596,6 +596,43 @@ <h2>On behalf of organisations</h2>
+				
+				<p>Tymczasem AMP zatrzymuje użytkowników wewnątrz domeny Google oraz przechwytuje ruch z innych stron internetowych na rzecz Google. W skali miliardów użytkowników jest to działanie, które <strong>jeszcze bardziej umocni dominującą pozycję Google w sieci internetowej</strong>.</p>
+				
+				<p>Zdajemy sobie sprawę z problemu powolnego ładowania się witryn internetowych, zwłaszcza w porównaniu do własnościowych rozwiązań takich jak <a href='https://instantarticles.fb.com/'>Facebook Instant Articles</a> i <a href='https://developer.apple.com/news-publisher/'>Apple News</a>. Wydawcy treści (zwłaszcza w sektorze mediów informacyjnych) od dawna zmagają się z trudnymi wyborami i niesłusznymi pobudkami, co prowadzi do złych decyzji i kompromisów, a te z kolei skutkują pogorszeniem wygody użytkowników.</p>
+				
+				<p>Wyszukiwarki to potężne narzędzia, które mają na tyle duże wpływy, żeby móc rozwiązać ten problem. Jednakże Google zdecydowało się stworzyć wyróżniającą się pozycję na samej górze wyników wyszukiwania (dla artykułów) oraz dodawać specjalną ikonę <i>błyskawicy</i> (dla pozostałych treści), ale tylko dla wydawców korzystających z technologii kontrolowanej przez Google, udostępnianej przez Google w ramach ich infrastruktury, w ramach adresu URL Google oraz wewnątrz interfejsu stworzonego przez Google (<a href='https://developers.google.com/search/docs/guides/about-amp'>źródło</a>).</p>
+				
+				<p>Idea AMP nie jest sama w sobie problematyczna, ale dwa aspekty jej implementacji umacniają Google na pozycji <i>de facto</i> standardowej platformy udostępniającej treść w sieci internetowej, zwłaszcza że Google próbuje zwiększyć użycie AMP:</p>
+				
+				<ol>
+					<li>Treść, która dostępna jest wewnątrz AMP (a tym samym wewnątrz domeny Google), otrzymuje dodatkową promocję w ramach wyników wyszukiwania, w tym: najwyższą pozycję dla artykułów informacyjnych.</li>
+					<li>Kiedy użytkownik decyduje się na odwiedzenie witryny z treścią promowaną przez Google, bezwolnie pozostaje wewnątrz ekosystemu Google.</li>
+
+				</ol>
+				
+				<p>Jeżeli poprawa wygody użytkowników sieci internetowej jest prawdziwym celem AMP, pragniemy zaproponować kilka prostych zmian, które na to pozwolą, a jednocześnie nie zagrożą dynamice i konkurencyjności sieci internetowej oraz zapewnią, że pozostanie skierowana przede wszystkim na potrzeby użytkownikaa:</p>
+				
+				<ol>
+					<li>Zamiast w szczególny sposób promować w ramach wyników wyszukiwania witryny internetowe wykorzystujące AMP, zacznijcie w taki sam sposób traktować wszystkie strony, które będą spełniać wybrane, obiektywne i neutralne kryterium wydajności, takie jak <a href='https://sites.google.com/a/webpagetest.org/docs/using-webpagetest/metrics/speed-index'>Speed Index</a>. Wówczas wydawcy treści będą mogli skorzystać z dowolnie wybranego przez nich rozwiązania technicznego.</li>
```

Comment:
> wybrane, obiektywne i neutralne kryterium wydajności

I seriously wonder if comma should be between "wybrane" and "obiektywne".

Reply:
There should be not.